backend/spellchecker/server/server.py

```diff
@@ -0,0 +1,45 @@
+# Author: Sergey Sokolov
+import time
+
+import grpc
+from concurrent import futures
+
+from spellchecker import Spellchecker
+import spellchecker_pb2
+
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+class SpellcheckerServicer(spellchecker_pb2.SpellcheckServicer):
+    """
+    gRPC service to check text that comes from stubs.
```

Comment:
it is service client who sends you the text, not a stub. Stub is a purely technical thing which provides typed interface on the client side.